Library/Formula/ledger.rb

```diff
@@ -20,7 +18,8 @@ class Ledger < Formula
 
   head "https://github.com/ledger/ledger.git"
 
-  option "debug", "Build with debugging symbols enabled"
+  option "with-debug", "Build with debugging symbols enabled"
+  option "with-check", "Run build-time check"
```

Comment:
Let's do this by default and add an option to turn it off (unless it takes a long time)